src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -15,9 +15,32 @@
 using Microsoft.Oryx.BuildScriptGenerator.Resources;
 using Microsoft.Oryx.Common.Extensions;
 using Microsoft.Oryx.Detector;
+using YamlDotNet.Serialization;
```

Comment:
@tulikac please merge with `main` and add integration tests

Reply:
Added some integration tests